### ISSUE TITLE
CA1822 - asp.net core configure method

### ISSUE THREAD

**Issue by dotnetshadow** [dotnet/roslyn-analyzers#2740]
#### Analyzer package

[Microsoft.CodeAnalysis.FxCopAnalyzers](https://www.nuget.org/packages/Microsoft.CodeAnalysis.FxCopAnalyzers)

#### Package Version
Example: v2.9.4 (Latest)

#### Diagnostic ID
[CA1822](https://docs.microsoft.com/en-us/visualstudio/code-quality/ca1822-mark-members-as-static?view=vs-2019)

#### Repro steps

1. Create an asp.net core 3.0 preview7 application
2. In startup.cs find the following line
```cs
 public void Configure(IApplicationBuilder app, IWebHostEnvironment env)
```
3. It shows the error twice
```Severity	Code	Description	Project	File	Line	Suppression State
Warning	CA1822	Member Configure does not access instance data and can be marked as static (Shared in VisualBasic)
Severity	Code	Description	Project	File	Line	Suppression State
Warning	CA1822	Member Configure does not access instance data and can be marked as static (Shared in VisualBasic)
```
4. If you do add static, it still shows one error

Note: I tried to suppress the warning with the following, but while it removed one warning it still had another warning remaining
```cs
[System.Diagnostics.CodeAnalysis.SuppressMessage("Microsoft.Performance", "CA1822:MarkMembersAsStatic")]
```

#### Expected behavior
Not sure this method needs to be marked static?

#### Actual behavior
It actually outputs the result twice

```Severity	Code	Description	Project	File	Line	Suppression State
Warning	CA1822	Member Configure does not access instance data and can be marked as static (Shared in VisualBasic)
Severity	Code	Description	Project	File	Line	Suppression State
Warning	CA1822	Member Configure does not access instance data and can be marked as static (Shared in VisualBasic)
```

**Comment by mavasani** [comment]
I believe the second warning is coming from Build, while the first one is from Intellisense. This also explains why adding the suppression only removes the intellisense diagnostic. Doing a rebuild should remove the second one too.

Ideally, the IDE should de-dupe the identical diagnostics from build and intellisense, but it seems that was not the case here. I tried to repro using your repro steps, but was unable to do so. Can you please retry on the latest VS2019 release: https://docs.microsoft.com/en-us/visualstudio/releases/2019/release-notes-preview
